services/QuillCMS/app/controllers/responses_controller.rb

```diff
@@ -72,6 +72,15 @@ def responses_for_question
     render json: @responses
   end
 
+  def multiple_choice_options
+    multiple_choice_options = Rails.cache.fetch("questions/#{params[:question_uid]}/multiple_choice_options", :expires_in => 900) do
+      optimal_responses = Response.where(question_uid: "-KQS6-2JxSzGYB2MERYc", optimal: true).order('count DESC').limit(2).to_a
```

Comment:
Should this hard-coded `question_uid` be a reference to `:question_uid` rather than the hard-coded value?